Header header
sensor_msgs/CameraInfo camera_info
int64 xmin
int64 ymin
int64 xmax
int64 ymax
int16 id
float64 probability
string Class
